README.md

```diff
@@ -35,11 +44,12 @@ Once you've got the data in, define views and perform reads via the PostgreSQL p
 
 Materialize supports a comprehensive variety of SQL features, all using the PostgreSQL dialect and protocol:
 
--   Joins, Joins, Joins! Materialize supports multi-column join conditions, multi-way joins, self-joins, cross-joins, inner joins, outer joins, etc.
+-   Joins, joins, joins! Materialize supports multi-column join conditions, multi-way joins, self-joins, cross-joins, inner joins, outer joins, etc.
 -   Delta-joins avoid intermediate state blowup compared to systems that can only plan nested binary joins - tested on joins of up to 64 relations.
 -   Support for subqueries. Materialize's SQL optimizer performs subquery decorrelation out-of-the-box, avoiding the need to manually rewrite subqueries into joins.
--   Materialize supports streams that contain CDC data (currently supporting the [Debezium](https://debezium.io/blog/2017/09/25/streaming-to-another-database/) format). Materialize can incrementally maintain views in the presence of arbitrary inserts, updates, and deletes. No asterisks.
--   All the aggregations. `GROUP BY` , `MIN`, `MAX`, `COUNT`, `SUM`, `STDDEV`, `HAVING`, etc.
+-   Materialize supports streams that contain CDC data (e.g., [Debezium](https://debezium.io/blog/2017/09/25/streaming-to-another-database/), [Striim](https://striim.com)). Materialize can incrementally maintain views in the presence of arbitrary inserts, updates, and deletes. No asterisks.
```

Comment:
```suggestion
-   Materialize supports streams that contain CDC data (e.g., [Debezium](https://debezium.io/blog/2017/09/25/streaming-to-another-database/), [direct PostgreSQL CDC](https://materialize.com/docs/sql/create-source/postgres/)). Materialize can incrementally maintain views in the presence of arbitrary inserts, updates, and deletes. No asterisks.
```

Reply:
Maybe we just strike the first sentence completely? The second sentence is the real point ("Materialize can incrementally maintain views in the presence of arbitrary inserts, updates, and deletes.") and is punchier without the first sentence. The paragraph above the list also already mentions the ways you can get data into Materialize these days (Kafka, PostgreSQL, webhooks).